pkg/minikube/constants/constants.go

```diff
@@ -35,6 +35,7 @@ const (
 	DefaultCPUS                  = 2
 	DefaultDiskSize              = "20g"
 	UpdateMarkerFileName         = "updated"
+	DefaultISO                   = ""
```

Comment:
The risk we have with this is that a constant seems to need a value... and when they give it one, it fails the intended function. Better to leave it as a local named variable